pyproject.toml

```diff
@@ -54,7 +54,7 @@ pygsheets = "^2.0.4"
 PyYAML = "^6.0.0"
 rdflib = "^6.0.0"
 setuptools = "^66.0.0"
-synapseclient = "4.5.1"
+synapseclient = "4.6.0"
```

Comment:
This is including a bug fix that we put out a while ago with the 4.5.1 version that had broken downloads from google buckets.